src/styles.scss

```diff
@@ -38,7 +38,7 @@ td {
     bottom: 0;
     left: 0;
     right: 0;
-    padding-top: 40%;
+    padding-top: 2 em;
```

Comment:
Let's pick where we're using ems or px on this site and use them consistently. Currently, it's mostly px. I don't mind going with ems. @JonathanKeslin, do you have a preference?